core/services/ardupilot_manager/main.py

```diff
@@ -28,13 +41,45 @@ def check_running_as_root() -> None:
         if os.geteuid() != 0:
             raise RuntimeError("ArduPilot manager needs to run with root privilege.")
 
-    @staticmethod
-    def start_navigator() -> None:
-        raise NotImplementedError("We only support ArduPilot running on serial devices.")
+    def start_navigator(self) -> None:
+        firmware = os.path.join(self.firmware_path, "ardusub")
```

Comment:
Can we todo to support other vehicles?

Reply:
This information will be provided by our endpoint, and the download will be done by manifest. Since we don't have both right now I'm hardcoding the values, but our python library already handle any vehicle and hardware.